system/boot.toit

```diff
@@ -28,7 +31,10 @@ Initialize the system and create the all important $ContainerManager
 */
 initialize -> ContainerManager:
   flash_registry ::= FlashRegistry.scan
-  service_manager := SystemServiceManager
+  service_manager ::= SystemServiceManager
+  network/NetworkServiceDefinition ::= platform == PLATFORM_FREERTOS
```

Comment:
Is the variable then used for anything?

Reply:
Not really, but I think I will rework it a bit soon.